pkg/common/parameters.go

```diff
@@ -265,3 +289,15 @@ func ExtractAndDefaultSnapshotParameters(parameters map[string]string, driverNam
 	}
 	return p, nil
 }
+
+func extractResorceTagsParameter(tagsString string, resourceTags map[string]string) error {
```

Comment:
type in function name

s/extractResorceTagsParameter/extractResourceTagsParameter

Reply:
@pwschuurman fixed the typo in the function name.